fastapi/applications.py

```diff
@@ -105,8 +105,9 @@ def setup(self) -> None:
             async def openapi(req: Request) -> JSONResponse:
                 return JSONResponse(self.openapi())
 
-            self.add_route(self.openapi_url, openapi, include_in_schema=False)
             openapi_url = self.openapi_prefix + self.openapi_url
+            self.add_route(openapi_url, openapi, include_in_schema=False)
```

Comment:
I doubt it's a bug? or it's just a prefix will affect on the swagger or redoc at begin of the design

Reply:
Yes, I doubt it's a bug too.
If not a bug, the kwargs `openapi_prefix` has some ambiguity.
And i'm not familiar with openapi, so i can't make sure my fix is right way.
Need some review.